contents/handbook/company/management.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Management at PostHog
+sidebar: Handbook
+showTitle: true
+---
+
+As we grow, we'll increase the number of managers at PostHog. Here's what a manager at PostHog looks like.
+
+## Defining the role of manager
+
+A manager at PostHog has these tasks
+1. Making sure their direct reports are happy and productive
+1. Setting the right context for direct reports to do their job
```

Comment:
Q related to this: What will the relationship between a manager and managee be? Will they be working on related projects? Will they be forming 'teams'?

If not, setting context might become hard to achieve as it's hard to provide right context for things you aren't working on yourself.

Reply:
Yeah this is a good question. Really depends on if we organize functionally or by product. Generally you organize functionally with dotted lines drawn to product orgs that you are working on. With some product manager responsible for the product context work.

At this point I don't think it's a huge issue since we are so small. I would expect that organizing functionally would effectively be organizing by product and the EM/TLM would be acting to some extent as the product manager until there is a dedicated PM to help out. Then the EM/TLM's job becomes more of a collaborative product context building with PM and engineering growth/teaching/happiness focus.